Makefile

```diff
@@ -22,8 +22,8 @@ INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
 BUILDROOT_BRANCH ?= 2017.02
 REGISTRY?=gcr.io/k8s-minikube
 
-HYPERKIT_BUILD_IMAGE 	?= karalabe/xgo-1.8.3
-BUILD_IMAGE 	?= gcr.io/google_containers/kube-cross:v1.8.3-1
+HYPERKIT_BUILD_IMAGE 	?= karalabe/xgo-1.9
+BUILD_IMAGE 	?= gcr.io/google_containers/kube-cross:v1.9
```

Comment:
Not sure how these are actually versioned, but you might want to try these kube-cross tags

```bash
$ gcloud container images list-tags gcr.io/google_containers/kube-cross
DIGEST        TAGS             TIMESTAMP
058ef002ff6a  v1.9.1-1         2017-10-16T14:23:55
7de0c648dfc1  v1.8.3-3         2017-10-02T17:09:08
37e2cf0ec470                   2017-10-02T16:38:16
890905b75b63  v1.8.3-2         2017-07-13T15:53:26
bf9d8c19414c  v1.7.6-k8s1.6-0  2017-05-25T11:49:37
9fda5e764ad7  v1.7.6-k8s1.5-0  2017-05-25T10:49:25
762d8f636782  v1.8.3-1         2017-05-25T02:31:12
ac1327a97b1b  v1.8.1-2         2017-05-16T04:30:25
fc8102dcf9ad                   2017-05-16T01:40:27
9805015b6c75  v1.8.1-1         2017-04-17T15:47:40
```

Reply:
Thanks! Also, I just checked and there actually isn't an image in 1.9 for the hyperkit build image, so should we only update the build image or leave them both at 1.8.3?

Reply:
There's a PR open for 1.9.1 for karalabe/xgo
https://github.com/karalabe/xgo/pull/92

I think its fine to just update the minikube ones (BUILD_IMAGE) for now - those are the ones that might be a little tricky with the vendoring.  

There should be no problem with building the hyperkit driver with 1.8 and minikube with 1.9 for a bit.